binaryclassifier/EmulatorMain.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 23 19:26:17 2016
+
+@author: critter
+
+From https://www.noao.edu/meetings/eventful-universe/Thursday0318/morning/CRTS_TucsonMar10.pdf :
+Discovery rate ~ 1 transient per 10^6 sources detected per epoch So, transient discovery seems very rare. 
+To spice things up, I've made transients appear much more often than what is described above (1 in 50)
+The frequency can be easily changed
+ 
+Each object contains a semi-random number of observations (2-10). Observations come in tuples of (time (seconds), light (no idea)). 
+Objects are arrays of these tuples where the first index is the first observation taken.
+"""
+
+from collections import namedtuple
+import random
+
+class EmulatorMain():
+    
+    def generateSingleObject(self):
+        """
+        Method to be called by the event broker or emulator when it wants
+        a single object's observational data. Returns an array of namedtuple.
+        if you wanted to emulate a real time event stream, it should return a
+        single namedtuple every x seconds instead of a whole array at once
+        """
+        totalObservations = random.randint(2, 10) #2 to 10 observations per object
+        y = random.randint(0, 50) ##If y = 0, it will generate as a transient
+        x = 0
+        initialBrightness = random.randint(1, 100) #arbitrary value for the brightness observed
+        isSlopePositive = random.choice([True, False])
+        hasSlopeChanged = False
+        if y == 0:
+            variance = random.randint(1, (initialBrightness/4) + 1)
+        timeTaken = 0 # pretend in seconds
+        timeInterval = random.randint(3600, 28800)
+        #observations may take place between 1 and 8 hours but remain constant for the object
+        FinalObject = []
+        while x < totalObservations:
+            Point = namedtuple('Observation' + str(x), ['time', 'light'])
+            if (y != 0):
+                timeTaken = timeTaken + timeInterval
+                p = Point(timeTaken, (initialBrightness + self._createError(initialBrightness)))
+                FinalObject.append(p)
+                x += 1
+                continue
+            if (hasSlopeChanged == False):
+                j = random.randint(1, 20) # 1 in 20 chance of slope changing
+                if (j == 1):
+                    hasSlopeChanged = True
+                    if isSlopePositive == False:
+                        isSlopePositive = True
+                    if isSlopePositive == True:
+                        isSlopePositive = False
+            if (isSlopePositive == True):
+                initialBrightness = initialBrightness + variance
+            if (isSlopePositive == False):
```

Comment:
You could get rid of the if condition here and just use an else statement instead.